src/crate/client/connection.py

```diff
@@ -28,16 +28,27 @@
 
 class Connection(object):
 
-    def __init__(self, servers=None, timeout=None, backoff_factor=0, client=None,
-                 verify_ssl_cert=False, ca_cert=None, error_trace=False,
-                 cert_file=None, key_file=None, username=None, password=None,
+    def __init__(self,
+                 servers=None,
+                 timeout=None,
+                 backoff_factor=0,
+                 poolsize=10,
```

Comment:
Could you move `poolsize` to the last position to not break BWC with calls that use positional arguments?

Reply:
Sure. Done.